databricks/koalas/frame.py

```diff
@@ -360,6 +360,19 @@ def columns(self, names):
         self._sdf = sdf
         self._metadata = self._metadata.copy(column_fields=names)
 
+    @property
+    def dtypes(self):
+        """Return the dtypes in the DataFrame.
+
+        This returns a Series with the data type of each column. The result's index is the original
+        DataFrame's columns. Columns with mixed types are stored with the object dtype.
+
+        :return: :class:`pd.Series` The data type of each column.
+        """
+        return pd.Series([to_arrow_type(self._sdf.schema[col].dataType).to_pandas_dtype()
```

Comment:
is there any reason for not calling directly the `.dtype` of the Series? You just added the same logic down there in `series.py`.